include/mbedtls/sha1.h

```diff
@@ -78,28 +83,114 @@ void mbedtls_sha1_clone( mbedtls_sha1_context *dst,
  * \brief          SHA-1 context setup
  *
  * \param ctx      context to be initialized
+ *
+ * \return         0 if successful
+ */
+int mbedtls_sha1_starts_ext( mbedtls_sha1_context *ctx );
+
+/**
+ * \brief          SHA-1 process buffer
+ *
+ * \param ctx      SHA-1 context
+ * \param input    buffer holding the  data
+ * \param ilen     length of the input data
```

Comment:
Just noticed that this is already being taken care of in https://github.com/ARMmbed/mbedtls/pull/1003.